### ISSUE TITLE
Timeout on trying to claim another box after reveal sequence

### ISSUE THREAD

**Issue by 70B43r** [netniV/stfc-mod#19]
![2024-02-12 21_46_22-STFC](https://github.com/tashcan/bob/assets/5332992/510ee0ee-4770-4dd8-8182-22a5e74c2e26)
![2024-02-12 21_46_53-STFC 2](https://github.com/tashcan/bob/assets/5332992/4ab64894-0ae4-45d3-b0d5-51a6373499de)
![2024-02-12 21_48_46-STFC 3](https://github.com/tashcan/bob/assets/5332992/4f273b43-8912-4722-a5d2-240e21b7ee2e)

I'm on Version 0.5.4 and happens since update.

**Comment by netniV** [comment]
That is a known issue scopely introduced when they add the extra button and did some weird behind the scenes stuff....

**Comment by tashcan** [comment]
Does it make sense to just remove that entire thing in the patch again, the functionality is in the game now, no? @netniV

**Comment by tashcan** [comment]
Wait. Really? I thought the issue was with the stay in bundle code?

**Comment by netniV** [comment]
No, I tested this and found that for both methods, and only when you turn off Reveal Screen does it work as intended by scopely.  

I have multiple ideas on why, but hitting escape makes it work so my primary one is this suggests there is a scene being dismissed that's not visible, which then makes the rest carry on.  By skipping the reveal sequence, maybe a visibility value is being missed and thus causes the issue?

Note, it doesn't call the doSkipReveal function when the extra button is pressed.

**Comment by tashcan** [comment]
Great. Scopely not even implementing things correctly according to their internal functions. Fun if someone ever decides to do skipreveal which is a feature in their whole UI thing...bunch of rubbish
